data/editor.config

```diff
@@ -260,7 +262,7 @@
         <include group="poi_internet" />
         <include field="cuisine" />
       </type>
-      <type id="amenity-ferry_terminal" can_add="no">
+      <type id="amenity-ferry_terminal">
```

Comment:
Aren't most of ferry terminals already mapped?

Reply:
there are many smaller terminals, e.g. it could be just a place from where small leisure boat rides are offered to tourists; in my experience all such places are tagged as "ferry_terminal"